packages/components/src/styles/hidden-links/hidden-links.stories.ts

```diff
@@ -121,7 +65,10 @@ export const StackLinks = {
 };
 
 /**
- * Use the `sd-hidden-links--multiple` modifier to show mutiple `sd-navigation-item` elements. (Recommended for >2 links)
+ * Use the `&--multiple` class to show mutiple `sd-navigation-item` elements.
```

Comment:
For the sample the links should be called:

Hidden Link 1
Hidden Link 2
Hidden Link 3

Reply:
Same for: Title For Multiple Links

Reply:
Updated on german level as well